pkg/config/app/kuma-cp/kuma-cp.defaults.yaml

```diff
@@ -29,6 +29,9 @@ store:
     # Maximum number of open connections to the database
     # `0` value means number of open connections is unlimited
     maxOpenConnections: 0 # ENV: KUMA_STORE_POSTGRES_MAX_OPEN_CONNECTIONS
+    # Maximum number of connections to the idle connection pool
+    # <0 value means no idle connections and 0 means default max idle connections
```

Comment:
how about `negative value means that the idle connection will be closed immediately. 0 means that no idle connection will be closed`

Reply:
0 means max idle connections would be default value which is 2 (const defaultMaxIdleConns = 2). 
And yes for negative values, he idle connection will be closed immediately